src/views/image/IsoZones.vue

```diff
@@ -32,15 +32,15 @@
       <template slot="action" slot-scope="text, record">
         <span style="margin-right: 5px">
           <a-button
-            :disabled="!('copyIso' in $store.getters.apis)"
+            :disabled="!('copyIso' in $store.getters.apis && record.isready)"
             icon="copy"
             shape="circle"
             :loading="copyLoading"
             @click="showCopyIso(record)" />
         </span>
         <span style="margin-right: 5px">
           <a-popconfirm
-            v-if="'deleteIso' in $store.getters.apis"
+            v-if="'deleteIso' in $store.getters.apis && record.isready"
```

Comment:
@Pearl1594 can you check against old UI, if deleting should be allowed? For example for error on download or uploading of local file?